adds some new ideas and probable rarity for them

kappa-sama opened the pull request:
also honestly we probably shouldn't have x-ray eyes for the organs, doesn't really make sense that they'd develop xray and also feels like it'd lead to too much "ooga powergame yeyeye"

and a lot of our positive cells should have clear drawbacks or restrictions that let them be powerful without being op, like the dwarf heart noclone

Commit:
adds some new ideas and probable rarity for them

also honestly we probably shouldn't have x-ray eyes

ghost commented:
also maybe we should have organs or organ traits made with cell combat in mind as their primary use, so we could have full on pokemon tournaments in medbay where people bring their strongest combat culture to fight other ones

Commit:
cargo crate

Commit:
some trait ideas and a "misc" idea section

Commit:
i'm braindead right now

i'll try to make an actual organ tomorrow after i hit that peak performance where i'm tired enough to do work without thinking about it but not so tired that i can't see

Commit:
ouch oof my bones

Thalpy commented:
Oh! Heck! When did this get here?

Thalpy committed:
Merge branch 'Tamigorgans' into patch-1
